src/widgets/sceneviewwidget.cpp

```diff
@@ -314,7 +310,7 @@ void SceneViewWidget::doObjectPicking()
     emit sceneNodeSelected(hitList.last().hitNode);
 }
 
-void SceneViewWidget::doPicking(const QSharedPointer<iris::SceneNode>& sceneNode,const QVector3D& segStart,const QVector3D& segEnd,QList<PickingResult>& hitList)
+void SceneViewWidget::doMeshPicking(const QSharedPointer<iris::SceneNode>& sceneNode,const QVector3D& segStart,const QVector3D& segEnd,QList<PickingResult>& hitList)
```

Comment:
Meshes and lights are considered viewport entities but i created different functions for picking either because their picking methods are different.
what would are you using a function called doMeshPicking for?